VideoLocker/src/main/java/org/edx/mobile/view/CourseDiscussionCommentsFragment.java

```diff
@@ -41,6 +44,9 @@
     @Inject
     Context context;
 
+    @InjectExtra(Router.EXTRA_DISCUSSION_TOPIC_IS_CLOSED)
+    boolean isTopicClosed;
```

Comment:
`topic` or `thread` they both translate to same construct i guess, so why rename it ?